unit/src/main/include/config/constants.h

```diff
@@ -4,11 +4,25 @@
 #include <opencv2/opencv.hpp>
 #include <opencv2/aruco/dictionary.hpp>
 #include <opencv2/aruco.hpp>
+#include <string>
 
 // TODO: Need a source file?
 // TODO: doesnt follow clang style guide
 
 namespace constants {
+    namespace general {
+        const int kNumMeasurements = 8;
+        enum Measurements {
```

Comment:
What is kNumMeasurements looks a bit strange. Minimize the number of places you need to change code when you are adding or removing a variable from being uploaded. Also, does not look like it belongs in the general namespace

Reply:
This belongs in the general namespace because you are gonna read and write to these measurements throughout the entire program. You are pulling data from sensors and putting it into the array, which uses this measurements enum. The measurements enum is there so that you don't have to keep checking back at which order the array is in (for example if I want to upload the ph, I would have to look back into other to code find with index the ph is stored in). The enum makes this way better because then I can just enter the name as the array index, and I don't have to worry about anything else.

kNumMeasurements is there to initialize the other arrays. The other way to do this is to add a dummy enum at the end to count the number of items in the enum, but this may cause confusion.

The reason this is done this way is because if I use a map/dictionary, there is no way for me to check if I am writing to a wrong key. This enum prevents this from happening and makes sure that the data goes in the correct place. This minimizes the amount of error.